test/io/test_load_image.py

```diff
@@ -22,7 +22,7 @@ def available_package() -> bool:
 
 
 def create_random_img8(height: int, width: int, channels: int) -> np.ndarray:
-    return (np.random.rand(height, width, channels) * 255).astype(np.uint8)
+    return (np.random.rand(height, width, channels) * 255).astype(np.uint8)  # noqa: NPY002
```

Comment:
i think this should works
```suggestion
    return np.random.uniform(low=0, high=255, size=(height, width, channels)).astype(np.uint8)
```

Reply:
Yes